.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: python
+python: 2.6
+addons:
+  apt:
+    packages:
+      # 'lockfile' command is provided by procmail package (yes...)
+      - procmail
+env:
+    - DMTCP_VERSION=2.5.1
+before_install:
+    # download & build DMTCP
+    - wget https://github.com/dmtcp/dmtcp/archive/${DMTCP_VERSION}.tar.gz
+    - mkdir -p $PWD/DMTCP-${DMTCP_VERSION}
+    - tar xfz ${DMTCP_VERSION}.tar.gz -C $PWD/DMTCP-${DMTCP_VERSION} --strip-components 1
+    - cd DMTCP-${DMTCP_VERSION} && ./configure --prefix $TRAVIS_BUILD_DIR && make && make install && cd -
```

Comment:
run `make check`?

Reply:
well, I could, but we're not testing DMTCP here, we're testing `csub`?

Reply:
`make check` tests the install, if you don't want to install, should use a package

Reply:
No package available in Travis, otherwise I would have used it (although that does imply not controlling the version).

It makes the test time a lot longer (~90s -> ~360s just for installing DMTCP), but that's not a huge issue I guess...